pom.xml

```diff
@@ -59,21 +66,28 @@
 
     <build>
         <pluginManagement>
-          <plugins>
+            <plugins>
+                <plugin>
+                    <groupId>com.itemis.maven.plugins</groupId>
+                    <artifactId>unleash-maven-plugin</artifactId>
+                    <version>2.7.2</version>
+                    <dependencies>
+                        <dependency>
+                            <groupId>com.itemis.maven.plugins</groupId>
+                            <artifactId>unleash-scm-provider-git</artifactId>
+                            <version>2.1.0</version>
+                        </dependency>
+                    </dependencies>
+                </plugin>
+            </plugins>
+        </pluginManagement>
+        <plugins>
             <plugin>
-              <groupId>com.itemis.maven.plugins</groupId>
-              <artifactId>unleash-maven-plugin</artifactId>
-              <version>2.7.2</version>
-              <dependencies>
-                <dependency>
-                  <groupId>com.itemis.maven.plugins</groupId>
-                  <artifactId>unleash-scm-provider-git</artifactId>
-                  <version>2.1.0</version>
-                </dependency>
-              </dependencies>
+                <groupId>org.commonjava.maven.plugins</groupId>
+                <artifactId>directory-maven-plugin</artifactId>
+                <version>0.2</version>
```

Comment:
Isn't the version already defined in the pluginManagement section of `poms/pom.xml`?

Reply:
@maggu2810 Yes, I guess there is no point to repeat the version here. I updated the pull request.